src/services/wyreService.js

```diff
@@ -188,6 +188,32 @@ class WyreService {
       };
     }
   }
+  getRates = async () => {
+    try {
+      const data = await this.service.get('https://api.sendwyre.com/v3/rates')
+      return {
+        data: data.data,
```

Comment:
Look at other examples, you need to return the error and everything like the other api calls